.travis.yml

```diff
@@ -79,8 +79,7 @@ jobs:
 
     # Data API stack
     - env: TASK=snapshot_scheduler-test
-    - env: TASK=snapshot_convertor-test
-    - env: TASK=elasticdump-test
+    - env: TASK=snapshot_generator-test
```

Comment:
Don’t forget to update the cached `target` directory.

Reply:
Thanks, will do